firebase-ai/src/main/kotlin/com/google/firebase/ai/LiveGenerativeModel.kt

```diff
@@ -68,6 +69,7 @@ internal constructor(
     requestOptions: RequestOptions = RequestOptions(),
     appCheckTokenProvider: InteropAppCheckTokenProvider? = null,
     internalAuthProvider: InternalAuthProvider? = null,
+    generativeBackend: GenerativeBackend? = null,
```

Comment:
Is GenerativeBackend ever actually null? If possible, I'd prefer this be non-nullable, so that behavior is more explicit.